oci-usage-to-data-dog-metrics.py

```diff
@@ -59,6 +59,7 @@
 import json
 import logging
 import re
+import requests
 from fdk import response
```

Comment:
I had to remove this to get the script to work and I don't think it's used anywhere

Reply:
It's needed for the upload to DataDog, and defined in Requirements.txt hopefully that explains it.